src/services/services.ts

```diff
@@ -3740,7 +3760,16 @@ namespace ts {
                 }
             }
             if (symbolFlags & SymbolFlags.Class && !hasAddedSymbolInfo) {
-                displayParts.push(keywordPart(SyntaxKind.ClassKeyword));
+                // Special case for class expression because we would like the entry detail to indicate that

```

Comment:
Is this specific to entry details? I thought quick info used it to. I would just say "we would like to indicate that"